src/installer/tests/Assets/TestProjects/HostApiInvokerApp/HostFXR.cs

```diff
@@ -209,17 +209,20 @@ static void Test_hostfxr_get_dotnet_environment_info(string[] args)
 
                 hostfxr_version = environment_info.hostfxr_version;
                 hostfxr_commit_hash = environment_info.hostfxr_commit_hash;
-                    
+
                 int env_info_size = Marshal.SizeOf(environment_info);
-                if (env_info_size != environment_info.size)
+
+                if ((IntPtr.Size == 4 && (env_info_size != (uint)environment_info.size)) ||
```

Comment:
`IntPtr.Size == 4` Do we ever run our tests in 32-bit? Also, this is calling into itself, right? If so I don't see why this is needed.